src/components/UserManagement/UserTableData.jsx

```diff
@@ -77,7 +78,9 @@ const UserTableData = React.memo((props) => {
       <td>{props.user.endDate ? props.user.endDate.toLocaleString().split('T')[0] : 'N/A'}</td>
       <td>
         <span className="usermanagement-actions-cell">
-          <button
+          {props.user.role === 'Owner' && !hasPermission(props.role, 'addDeleteEditOwners') 
```

Comment:
Could you explain why this is a `owner` only operation?

Reply:
Only Owners can add/delete/edit other owner profiles!